### PR TITLE
Learn more component

### PR DESCRIPTION
**Pull request by Mickaellah**
Hello everyone,

Team Dong has just finished the learn more banner component. Please have a look at it whenever it is possible.
Do not hesitate to leave some comments.

Ticket: https://app.clickup.com/t/45e4mb.
Branch: https://github.com/onja-org/vamalaka-dong/tree/learn-more.

Cheers,
Team Dong

### CODE REVIEW

**Review by Tanosimboangy — APPROVED** [review]

File: src/components/LearnMore/LearnMore.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        button {
+            font-size: 18px;
+        }
+    }
+`;
+
+export interface LearnMoreProps {
+    learnmoreheading: string,
+    learnmoredescription: string,
+    isBlueButton?: boolean
```
Comment (andrzej-onja) [inline src/components/LearnMore/LearnMore.tsx]: rename to `isPrimary`

File: src/components/Buttons/Buttons.tsx [hunk 2]
```diff
@@ -1,14 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
 import { fonts } from '../../globalStyles/fonts'
-
 export interface ButtonsProps {
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx]: rename to `ButtonProps`

File: src/components/Buttons/Buttons.tsx [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
   label?: string
   learnEnabled?: boolean
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx]: isPrimary

File: src/components/Buttons/Buttons.tsx [hunk 4]
```diff
@@ -19,7 +17,6 @@ const SecondWrapper = styled.div`
 `
 const Button = styled.button<ButtonsProps>`
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx]: ButtonStyled

File: src/components/Buttons/Buttons.tsx [hunk 5]
```diff
@@ -63,4 +59,4 @@ const Buttons: React.FC<ButtonsProps> = ({
     </FirstWrapper>
   )
 }
-export default Buttons
+export default Buttons
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx]: Button

**Review by andrzej-onja — APPROVED** [review]
Please address the comments

**Review by Ny-Rianala — APPROVED** [review]

**Pull request merged**